sections/introduction.tex

```diff
@@ -1,5 +1,25 @@
 \section{Introduction}
 %% Argue briefly for the relevance of the studied area/problem — start from general and end with your concrete problem.
+A general challenge for search engines, no matter the environment, is how to acquire relevant search results from a given corpus.
+While some search engines use simple algorithms, such as finding all articles including a given search term and sorting by date, this may not always give useful results.
+Because of this we want to explore ways to improve search results while using information already available from a corpus.
+
+A way to improve searches could be by labeling documents in a corpus and using these labels to get results.
+A popular way to label documents is topic modeling.
+Topic modeling gives each document in a corpus a distribution of topics, which can then be used for different applications.
+More specifically, we look into \gls{lda} since it is currently the most used topic model, and is applicable for many situations\cite{lda}.
+
+Topic modeling does not by itself help with how to rank documents in a search, which is why an additional part is necessary.
+We have chosen to explore a popular way to rank documents, which is by using PageRank.
+PageRank is an algorithm originally meant for ranking web pages in search engine results by their importance, but it can easily be extended to other types of data, such as news articles.
```

Comment:
```suggestion
PageRank is an algorithm originally used for ranking web pages in search engines, but it can easily be extended to other types of data, such as news articles.
```